README.md

```diff
@@ -4,19 +4,35 @@
 
 > This a web app built on Rails API as backend and React framework as a frontend. This app is used for reserving a doctor's appointment by patients. Users can create, view, edit and delete a list of appointment(s).
 
-## Demo Video
+## Live Demo Link
```

Comment:
- [ ] On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

 ## Tips and extra help

Go [here] (https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪